lib/task/migrate/migrations/arMigration0180.class.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Access to Memory (AtoM) software.
+ *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ * Add new user interface label for actor relations.
+ *
+ * @package    AccesstoMemory
+ * @subpackage migration
+ */
+class arMigration0180
+{
+  const
+    VERSION = 180, // The new database version
+    MIN_MILESTONE = 2; // The minimum milestone required
+
+  /**
+   * Upgrade
+   *
+   * @return bool True if the upgrade succeeded, False otherwise
+   */
+  public function up($configuration)
+  {
+    if (null === QubitSetting::getByName('authority_record_relationships'))
+    {
+      $setting = new QubitSetting;
+      $setting->name  = 'Authority Record Relationships';
```

Comment:
This should be the the `$setting_i18n->value` value.  The name should match the name in the fixture: `authority_record_relationships`.

Reply:
D'oh! Thanks... should have tested this migration.